package.json

```diff
@@ -18,34 +15,47 @@
     "lint:fix": "eslint \"src/**/**.{ts,tsx}\" --fix",
     "perf": "parcel perf/index.html",
     "build:tsc": "tsc --project ./tsconfig.json",
-    "build": "npm run clean && npm run build:tsc && parcel build",
+    "build:umd": "parcel build --target legacy --dist-dir dist/legacy",
+    "build": "npm run clean && npm run build:tsc && npm run build:umd",
+    "copy": "cp package* README.md LICENSE dist/",
     "examples:dev": "npm run clean && parcel examples/index.html",
     "docs:dev": "npm run clean && parcel docs-src/index.html",
     "docs:build": "rm -rf ./docs/ && parcel build ./docs-src/index.html --dist-dir docs/ --public-url https://sayari-analytics.github.io/trellis/ && cp ./docs-src/assets/og-image.png ./docs/"
   },
+  "targets": {
+    "main": false,
+    "module": false,
+    "types": false,
+    "legacy": {
+      "context": "browser"
+    }
+  },
   "author": "James Conkling <james.lane.conkling@gmail.com> jameslaneconkling.github.io",
   "license": "ISC",
   "dependencies": {
-    "@pixi/unsafe-eval": "^6.5.5",
+    "@pixi/events": "^6.5.10",
+    "@pixi/unsafe-eval": "^6.5.10",
     "d3-color": "^3.1.0",
     "d3-force": "^3.0.0",
     "d3-hierarchy": "^3.1.2",
     "d3-interpolate": "^3.0.1",
     "fontfaceobserver": "^2.1.0",
-    "pixi.js-legacy": "^6.5.5"
+    "parcel": "^2.9.3",
+    "pixi.js": "^7.2.4",
```

Comment:
we should only ship with one version of pixi, probably pixi.js-legacy. though note that I haven't checked if any of my new renderer stuff works w/ Canvas... if it doesn't/can't, then publishing this w/ the pixi-legacy canvas fallback isn't worth it.